cfn-resources/alert-configuration/mongodb-atlas-alertconfiguration.json

```diff
@@ -152,35 +123,7 @@
         },
         "EventTypeName": {
           "type": "string",
-          "description": "Incident that triggered this alert.",
```

Comment:
Getting several warning from CFN and CDK-IMPORT saying that is not a good practise to keep big enum in the definition file of the resource as it is hard to maintain and decrease readability

Reply:
Okay it'll be bad UX though:(

Reply:
> Okay it'll be bad UX though:(

I don't think it would be better with the current enums. I personally would be overwhelmed by [this enum](https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/e889ca79a143cfebeff07dd69a071d13a3ce45b9/cfn-resources/alert-configuration/mongodb-atlas-alertconfiguration.json#L220) and I would use the documentation anyway to understand what is the purpose of each metric name

Reply:
### Old Approach

- Pros:
  - you can import the enum and get all the values. Example: `MetricThresholdViewMetricName.NORMALIZED_SYSTEM_CPU_USER`

- Cons:
  - the enum has a lot of entries
  - if a new value is added in the endpoint, the user won't be able to use it in the CDK until we update the CFN resource and update the CDK constructor
  - the resource definition is hard to read and maintain
 

## New approach
- Pros:
  - you can use a new value as soon as it is available in the API
  - the definition file is shorter and easy to read and maintain

- Cons:
  - users will have to check the documentation to see the values available for the enum

Reply:
it's a tradeoff, but as we scale i think the pain @andreaangiolillo highlighted below is only going to increase over time. also if we don't maintain well that could introduce another set of user friction. suggest we go with this approach for now and keep eyes/ears open in case of any user concerns.  

> Getting several warnings from CFN cli and CDK-IMPORT saying that is not a good practice to keep big enum in the definition file of the resource as it is hard to maintain and decreases readability